pylintrc

```diff
@@ -123,7 +123,7 @@ enable=useless-suppression
 bad-functions=
 
 # Good variable names which should always be accepted, separated by a comma
-good-names=i,j,k,ex,Run,_
+good-names=i,j,k,e,f,m,cm,Run,_,n,op,it
```

Comment:
Might be just my personal preference, but I normally use `ex` over just `e`.

Reply:
Let's add both then :)